Makefile

```diff
@@ -74,5 +74,9 @@ update-spelling:
 	hack/make-rules/update/misspell.sh
 verify-spelling:
 	hack/make-rules/verify/misspell.sh
+update-go-deps:
+	hack/make-rules/update/go-deps.sh
+verify-go-deps:
+	hack/make-rules/verify/go-deps.sh
 #################################################################################
-.PHONY: unit test verify go-lint update-gofmt verify-gofmt update-spelling verify-spelling
+.PHONY: unit test verify go-lint update-gofmt verify-gofmt update-file-perms verify-file-perms update-spelling verify-spelling update-go-deps verify-go-deps
```

Comment:
BTW: it's also possible to split these up to avoid rebase hell (stick .PHONY: foo alongside foo:) but under normal circumstances I wouldn't expect this to change as often anyhow (after we finish migrating)

Reply:
good point, will do that if I ever need to add more